doc/prefix.mtree.in

```diff
@@ -1,6 +1,6 @@
 # MacPorts filesystem hierarchy, for internal use only. Changes to this file will not stick across installations.
 
-/set type=dir uname=@DSTUSR@ gname=@DSTGRP@ mode=@DSTMODE@
+/set type=dir uname=@DSTUSR@ gid=@DSTGID@ mode=@DSTMODE@
```

Comment:
This is OK for local installation, but the mtree files also get shipped in the binary packages. I don't think we can guarantee that the gid will refer to the same user on the destination machine as on the build machine.

Reply:
How does that work then with the group name? That has the same problem right? Or am I missing something?

Reply:
The user and group are recorded by name in the pkg.

Reply:
But the group name might be invalid on the destination machine too, or am I just completely misunderstanding this?

Reply:
We can be pretty sure there will be a group named `admin` on macOS systems.